include/dbclean.php

```diff
@@ -16,22 +16,48 @@
 load_config('config');
 load_config('system');
 
-update_shadow_copy();
+remove_orphans();
```

Comment:
Sorry for the formal gibberish. "Actions" isn't even the word used in the standards. Basically, what I called action is everything that is not declaring constants, variables, functions and classes.

This includes:
- generating output
- explicit use of `require` or `include` (usage of `use` is fine)
- connecting to external services
- modifying ini settings
- emitting errors or exceptions
- modifying global or static variables
- reading from or writing to a file
- ...